tests/cross/src/test/scala/tests/pc/ExtractMethodSuite.scala

```diff
@@ -0,0 +1,381 @@
+package tests.pc
+
+import java.net.URI
+
+import scala.meta.internal.jdk.CollectionConverters._
+import scala.meta.internal.metals.CompilerOffsetParams
+import scala.meta.internal.metals.TextEdits
+
+import munit.Location
+import munit.TestOptions
+import org.eclipse.{lsp4j => l}
+import tests.BaseCodeActionSuite
+
+class ExtractMethodSuite extends BaseCodeActionSuite {
+
+  checkEdit(
+    "simple-expr",
+    s"""|object A{
+        |  val b = 4
+        |  def method(i: Int) = i + 1
+        |  <@val a = <<123 + method(b)>>@>
+        |}""".stripMargin,
```

Comment:
```suggestion
    s"""|object A{
        |  val b = 4
        |  def method(i: Int) = i + 1
        |  @@val a = <<123 + method(b)>>
        |}""".stripMargin,
```
Does `<@` symbolises where to extract the range? Maybe we could just use a position with `@@`